docs/developer/binarywheels.rst

```diff
@@ -0,0 +1,94 @@
+Building binary wheels
+======================
+
+Several packages in the Zope Foundation repositories contain C code, which
+means they should be packaged and published as binary wheels alongside the
+standard egg archive. Building binary wheels has its pitfalls, though. If not
+done right they can cause application instability and crashes.
+
+
+.. warning::
+    Publishing binary wheels built in a local build environment is highly
+    discouraged! Every build environment is different, binary wheels made for
+    publication on PyPI should only be built in standardized and predictable
+    environments.
+
+To minimize problems we have automated binary wheel building using CI providers
+like Appveyor and GitHub Actions. Wheels are built and uploaded using a special
+GitHub account named ``zope.wheelbuilder``. These configurations and the PyPI
```

Comment:
```suggestion
PyPI account named ``zope.wheelbuilder``. These configurations and the PyPI
```

Or do I miss something here? There is no `zope.wheelbuilder` at GitHub: https://github.com/search?q=zope.wheelbuilder&type=users